README.md

```diff
@@ -2,7 +2,22 @@
 
 ## Starting the server:
 
-
 1. Open a terminal and go to the server folder. Make sure you have **pipenv** installed (`pip install pipenv`)
 2. Install the dependencies with `pipenv install`. This also createa a virtual environment, if there isn't one already
 3. Activate the virtual environment and start the app with `pipenv run flask run`
```

Comment:
Get into the shell with this: `pipenv shell`
Then you can do whatever you want (like flask run)